app/code/Magento/Backend/Api/Search/ItemsInterface.php

```diff
@@ -0,0 +1,23 @@
+<?php
+/**
+ *
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace Magento\Backend\Api\Search;
+
+use Magento\Backend\Model\Search\SearchCriteria;
+
+/**
+ * @api
+ */
+interface ItemsInterface
```

Comment:
@ishakhsuvarov do you have a suggestion for @tzyganu in this case? I could think that `ResultsInterface` and `getResults` would work well.